common/content/options.js

```diff
@@ -41,7 +41,7 @@ const Option = Class("Option", {
 
         // add no{option} variant of boolean {option} to this.names
         if (this.type == "boolean")
-            this.names = array([name, "no" + name] for (name in values(names))).flatten().__proto__;
+            this.names = util.Array.flatten(Array([name, "no" + name] for (name in values(names)))).__proto__;
```

Comment:
Okay. After some testing I would not recommend accepting this pull request :) I found out that the proposed line doesn't work. The error is gone, but boolean options (e.g. `autocomplete`) are not available. I also found out that the call to `array(..)` is not a typo, but references the `array(..)` function defined in `base.js`. Somehow my beta version of FF 44 (I _think_ it was b7 or b8) prevented the scope in which`options.js` executes from "seeing" the `array(..)` function. After an update to 44b everything seems to work just fine. I don't have time to examine this further. I assume it was a FF related glitch.